.github/dependabot.yml

```
@@ -10,6 +10,12 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+    allow:
+      - dependency-type: "direct"
+    ignore:
+      - dependency-name: "django"
```

In this project's requirements it has Django. pip is supposed to be case insensitive so not 100% sure if this will work or not. Can just update it now to be capitalized. 
```suggestion
      - dependency-name: "Django"
```